tooling/prometheus-rules/Makefile

```diff
@@ -0,0 +1,21 @@
+SHELL = /bin/bash
+
+# Define the binary name
+BINARY = prometheus-rules
+
+# Define the source files
+SOURCES = $(shell find . -name '*.go')
+
+# Build the binary
+$(BINARY): $(SOURCES) $(MAKEFILE_LIST)
+	go build -o $(BINARY) .
+
+# Clean the build artifacts
+clean:
+	rm -f $(BINARY)
+
+.PHONY: clean
+
+kubernetesControlPlane-prometheusRule.yaml:
```

Comment:
I checked this file into the repo so none of our builds need network connectivity / GitHub availability to function.